modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractGoCodegen.java

```diff
@@ -144,8 +144,12 @@ public String escapeReservedWord(String name) {
 
     @Override
     public String toVarName(String name) {
+    	
+    	if(name != null) {
+    		name = name.replaceAll("-", "_");
+    	}
         // replace - with _ e.g. created-at => created_at
-        name = sanitizeName(name.replaceAll("-", "_"));
+        name = sanitizeName(name);
```

Comment:
will sanitizeName() always return a non-null object?

Reply:
@macjohnny `super` will always return non `null`. If `name` is `null` it will return `ERROR_UNKNOWN`
This is a different issue tracked here #376

Reply:
ok, so then this should be fine

Reply:
I could update this.